buffalo/cmd/test.go

```diff
@@ -22,9 +22,12 @@ import (
 const vendorPattern = "/vendor/"
 
 var vendorRegex = regexp.MustCompile(vendorPattern)
+var skipSchemaLoad = false
 
 func init() {
 	decorate("test", testCmd)
+	testCmd.Flags().BoolVarP(&buildOptions.ExtractAssets, "skip-schema-load", "s", false, "skips loading the schema and instead runs migrations before running tests")
```

Comment:
I'm not a fan of the name of this flag. Any other options?

Reply:
Oh boy! ouch! going for it.

Reply:
@markbates  what about "forceMigrations" and -m --force-migrations ?